Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.mm

```diff
@@ -187,6 +187,15 @@
     setNeedsDisplay(IntRect(IntPoint(), expandedIntSize(m_parameters.size)));
 }
 
+DestinationColorSpace RemoteLayerBackingStore::colorSpace() const
+{
+#if HAVE(IOSURFACE_RGB10)
+    if (m_parameters.type == Type::IOSurface && m_parameters.deepColor)
+        return DestinationColorSpace { extendedSRGBColorSpaceRef() };
```

Comment:
If you are feeling motivated, probably makes sense to add a static on DestinationColorSpace for this, as its used in a few places now.